### PR TITLE
Update for Fastboot 1.0

### PR DESCRIPTION
**Pull request by devotox**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "ember-cli-new-version",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (knownasilya) [inline package.json]: Please don't bump the version 👍

File: package.json [hunk 2]
```diff
@@ -21,25 +21,24 @@
...
-    "ember-disable-proxy-controllers": "^1.0.1",
-    "ember-export-application-global": "^1.0.4",
-    "ember-getowner-polyfill": "1.0.1",
-    "ember-try": "^0.2.6"
+    "ember-disable-prototype-extensions": "^1.1.2",
+    "ember-export-application-global": "^2.0.0",
+    "ember-load-initializers": "^1.0.0",
+    "ember-resolver": "^4.2.1",
+    "ember-source": "~2.14.0",
+    "ember-welcome-page": "^3.1.1",
```
Comment (knownasilya) [inline package.json]: Could you also remove this dependency? Thanks!

File: addon/components/new-version-notifier/component.js [hunk 3]
```diff
@@ -37,6 +36,8 @@ export default Ember.Component.extend({
         clearTimeout(currentTimeout);
       }
 
+      if(!Ember.$) { return; }
```
Comment (knownasilya) [inline addon/components/new-version-notifier/component.js]: Also, could you make this a multiline if, just so it goes with the rest of the syntax?

**Comment by sethwebster** [comment]
Thanks all for keeping things going 👍

File: package.json (hunk 1 above)
Comment (devotox) [inline package.json]: Ha yes forgot to change this back. When testing the package lock kept causing a cached version to be installed so took drastic measures

**Pull request merged**